log/logger_test.go

```diff
@@ -124,7 +124,7 @@ func Test_Caller(t *testing.T) {
 	a, buffer, log := Setup(t)
 
 	log.Info().With(StackTrace).Logf("message")
-	a.Regexp(regexp.MustCompile(`caller_0=(.*?)(\/log\/logger_test\.go)`), buffer.String())
+	a.Regexp(regexp.MustCompile(`caller_\d=(.*?)(\/log\/logger_impl\.go)`), buffer.String())
```

Comment:
We don't want to announce the logger_impl.go file. We want to filter this out.

Reply:
This is where the filter is: https://github.com/moov-io/base/pull/102/files#diff-330686dc447b6523e8cbb02ae01e63ed53f3b214274ecc0da55c978965dda5ddR63